app/javascript/modules/listingDetails/ListingDetailsUnitAccordion.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react"
+import { ContentAccordion, StandardTable, t } from "@bloom-housing/ui-components"
+import { RailsListingUnits } from "../../api/types/rails/listings/RailsListingUnits"
+
+export interface UnitGroupType {
+  units: RailsListingUnits[]
+  availability: number
+  minSqFt: number
+  maxSqFt: number
+}
+
+export interface ListingDetailsUnitAccordionProps {
+  unitType: string
+  unitGroup: UnitGroupType
+}
+
+const TableHeaders = {
+  unit: "listings.features.unit",
+  area: "listings.features.area",
+  baths: "listings.features.baths",
+  floor: "listings.features.floor",
+  accessibility: "listings.features.accessibility",
+}
+
+const getTableData = (units: RailsListingUnits[]) =>
+  units.map((unit) => ({
+    unit: { content: <span className="font-semibold">{unit.Unit_Number}</span> },
+    area: {
+      content: (
+        <>
+          <span className="font-semibold">{unit.Unit_Square_Footage}</span>
+          {t("listings.features.sqft")}
+        </>
+      ),
+    },
+    baths: { content: <span className="font-semibold">{unit.Number_of_Bathrooms}</span> },
+    floor: { content: <span className="font-semibold">{unit.Unit_Floor}</span> },
+    accessibility: { content: <span className="font-semibold">TODO</span> },
```

Comment:
I wasn't sure the set of possible values here; pinning this as a reminder to ask the team

Reply:
I'd +1 that you covered them all! Looking at https://github.com/SFDigitalServices/sf-dahlia-web/blob/main/app/assets/javascripts/listings/components/listing/features-section.html.slim#L54